test/spec/view/toolbar/MinimizedWindows.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('CpsiMapview.view.toolbar.MinimizedWindows', function() {
+    describe('Basics', function() {
+        it('is defined', function() {
+            expect(CpsiMapview.view.toolbar.MinimizedWindows).not.to.be(undefined);
+        });
+
+        it('can be instantiated', function() {
+            var inst = Ext.create('CpsiMapview.view.toolbar.MinimizedWindows');
+            expect(inst).to.be.a(CpsiMapview.view.toolbar.MinimizedWindows);
+        });
+    });
+});
```

Comment:
Missing new line at the end  (sorry for being picky :see_no_evil:)

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/179/commits/0b106a127d97601d29ea31d149448e09874ec88e